main.go

```diff
@@ -104,11 +106,21 @@ func main() {
 				}
 			})
 
+			// Calculate the depth of certificate that needs to be processed
```

Comment:
Is this from the leaf up or from the root down? It's not super clear to me at first glance.

Reply:
It would be from leaf up since that is usually the cert in interest. This calls for comments in the code.. I'll add it too..